src/reducers/listsSlice.ts

```diff
@@ -35,7 +38,7 @@ export interface IListState {
   division: Array<IListItem>;
   subDivision: Array<IListItem>;
   responsibleZone: Array<IListItem>;
-  hashTags: Array<IListItem>;
+  hashTags: IHashTagsResponse;
```

Comment:
We need to create a new store for hashtags, since the hashtags are no longer of type IListItem.